### PR TITLE
chore(docs): explain the process for merging changes to master

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: COMMITTER.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Angular committers and also prevent breakages on master.
+
+When your PR is ready to merge, a project member in [this list] should add the special label,
+`PR: merge`.
+A robot running as [marypoppins] is notified that the label was added by an authorized person,
+and will create a new branch in the angular project, using the convention `presubmit-{username}-PR-{number}`.
+If the automation fails, committers can also push to a branch following this naming scheme.
+
+When a Travis build succeeds for a branch named `presubmit-*`, Travis will re-base the PR and merge to master
+automatically.
```
Comment (mhevery) [inline COMMITTER.md]: and closes the PR

File: TRIAGE_AND_LABELS.md [hunk 2]
```diff
@@ -46,8 +46,8 @@ Every triaged PR must have a `pr_action` label assigned to it and an assignee:
  
 * `pr_action: cleanup` -- more work is needed from the current assignee. 
 * `pr_action: discuss` -- discussion is needed, to be led by the current assignee.
-* `pr_action: merge` -- OK to merge this as soon as tests are green, `pr_state: LGTM`, and `CLA: 
-yes` are true. assignee (or anyone else) can merge.
+* `pr_action: merge` -- OK to merge this if the travis build is green, `pr_state: LGTM`, and `CLA:
```
Comment (mhevery) [inline TRIAGE_AND_LABELS.md]: I would change it to: `Add to a PR if you would like to trigger the automatic merging on successful build.` (Or something along those lines. Current text is confusing. ) I think the sentence needs to be more active.

File: COMMITTER.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+As a contributor, just read the instructions in [CONTRIBUTING.md](CONTRIBUTING.md) and send a pull request.
+Someone with committer access will do the rest.
+
+## The `PR: merge` label and `presubmit-*` branches
+
+We have automated the process for merging pull requests into master. Our goal is to minimize the disruption for
+Angular committers and also prevent breakages on master.
+
+When a PR is ready to merge, a project member in [this list] should add the special label,
```
Comment (alexeagle) [inline COMMITTER.md]: where is the list? how do we add people? looks like they are just stored in the app engine DB
https://github.com/mhevery/angular2-automation/blob/4606b6a4837a69a913d57eb8a6257d1199356821/helloworld.py#L15
but how do we see the list, where is the app hosted?

Comment (mhevery) [inline COMMITTER.md]: I don't know enought about AppEngine to know how to share the rights of the app with the wider group. Yes it is stored in DB

Comment (alexeagle) [inline COMMITTER.md]: Added enough of a pointer so someone can track this down when needed.

**Pull request closed without merging**